src/util/math-util.js

```diff
@@ -45,4 +45,15 @@ MathUtil.wrapClamp = function (n, min, max) {
     return n - Math.floor((n - min) / range) * range;
 };
 
+/**
+ * Convert a Scratch color number to a hex string, #RRGGBB.
+ * @param {number} color RGB color as a decimal.
+ * @return {string} RGB color as #RRGGBB hex string.
+ */
+MathUtil.scratchColorToHex = function (color) {
```

Comment:
I wonder if we might want to move this into a dedicated `color` utility module eventually. This should also have test coverage.